app/src/main/java/com/dessalines/thumbkey/ui/theme/Color.kt

```diff
@@ -525,5 +655,6 @@ fun blue(): Pair<ColorScheme, ColorScheme> {
         outlineVariant = md_theme_dark_outlineVariant,
         scrim = md_theme_dark_scrim,
     )
+    }
```

Comment:
The code didn't compile, probably because of additions like this. 

You also might need to run `./gradlew formatKotlin` to pass lint.